test/integration/integration.cc

```diff
@@ -322,7 +325,10 @@ uint32_t BaseIntegrationTest::lookupPort(const std::string& key) {
   if (it != port_map_.end()) {
     return it->second;
   }
-  RELEASE_ASSERT(false, "");
+  RELEASE_ASSERT(
```

Comment:
Would a `throw` make more sense?

Reply:
Nothing else in BaseIntegrationTest or anything above it throws; I will defer entirely to you on this, but my instinct is to stay consistent.